ocs_ci/ocs/constants.py

```diff
@@ -155,6 +155,11 @@
 PRIVILEGED = "privileged"
 ANYUID = "anyuid"
 
+
+CLUSTER_SERVICE_VERSION = "csv"
```

Comment:
You can remove this, I dont think you use CSV anywhere, instead you can replace with the resource name. 

if you want to use CSV there is already existing variable no need for a new one, https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L129

Reply:
Done